eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -62,6 +62,17 @@ case class Features(activated: Set[ActivatedFeature], unknown: Set[UnknownFeatur
 
   def hasPluginFeature(feature: UnknownFeature): Boolean = unknown.contains(feature)
 
+  def areSupported(remoteFeatures: Features): Boolean = {
+    // we allow unknown odd features (it's ok to be odd)
+    val unknownFeaturesOk = !remoteFeatures.unknown.exists(_.bitIndex % 2 == 0)
```

Comment:
rewritten in a positive way:

```suggestion
    val unknownFeaturesOk = remoteFeatures.unknown.forall(_.bitIndex % 2 == 1)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1576/commits/c026fa00a7c9e2223760850eae220f810829067b